pkg/operator/collection.go

```diff
@@ -66,6 +66,11 @@ func setupCollectionControllers(op *Operator) error {
 		namespace: op.opts.OperatorNamespace,
 		name:      NameCollector,
 	}
+	// Collector secret.
+	objFilterSecret := namespacedNamePredicate{
+		namespace: op.opts.OperatorNamespace,
+		name:      CollectionSecretName,
```

Comment:
does this limit the operator to just watching the collector secret in gmp-system? what about the others, such as the new alertmanager secret?

Reply:
Added.